lddtree.sh

```diff
@@ -96,11 +97,14 @@ elf_specs_binutils() {
 		| tr '\n' ' '
 }
 
+
+
+
 # elf wrapper functions
-elf_rpath() { elf_rpath_$BACKEND "$@"; }
-elf_interp() { elf_interp_$BACKEND "$@"; }
-elf_needed() { elf_needed_$BACKEND "$@"; }
-elf_specs() { elf_specs_$BACKEND "$1"; }
+elf_rpath() { [ ! -z "$1" ] && elf_rpath_$BACKEND "$@" | sed -e "s:[$]ORIGIN:${1%/*}:g" | sed -e "s:[$]{ORIGIN}:${1%/*}:g" | sed -e "s,:\.,:${1%/*},g" | sed -e "s,^\.,${1%/*},g"; }
```

Comment:
- g flag resolve case when rpath values contains several path with "ORIGIN"
eg:
$ORIGIN/lib64:/lib:$ORIGIN/lib32

- Right, one sed and 4 -e is much better !